### PR TITLE
Add frontegg scripts

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

File: frontegg/fusionAuthDockerFiles/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+version: '3'
```
Comment (ghost) [inline frontegg/fusionAuthDockerFiles/docker-compose.yml]: remove version
```suggestion

```

File: frontegg/fusionAuthDockerFiles/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    networks:
+      - db_net
+    volumes:
+      - db_data:/var/lib/postgresql/data
+
+  fusionauth:
+    image: fusionauth/fusionauth-app:1.48.3
```
Comment (ghost) [inline frontegg/fusionAuthDockerFiles/docker-compose.yml]:
```suggestion
    image: fusionauth/fusionauth-app:latest
```

File: frontegg/fusionAuthDockerFiles/docker-compose.yml
Comment (ghost) [inline frontegg/fusionAuthDockerFiles/docker-compose.yml:1]: We usually don't add container names anymore either as they have collisions if you have multiple floating around, happy to hear your thoughts?

**Review by ghost — CHANGES_REQUESTED** [review]
Small change to Dockerfile

File: frontegg/fusionAuthDockerFiles/docker-compose.yml
Comment (rideam) [reply frontegg/fusionAuthDockerFiles/docker-compose.yml:1]: Okay yes the names do cause collisions.  I have removed the container names.

Comment (rideam) [reply frontegg/fusionAuthDockerFiles/docker-compose.yml:1]: Though in other guides we do make use of the container names so its not always possible to provide the configuration without the container names

**Pull request merged**